packages/core/src/interaction-output.ts

```diff
@@ -68,11 +68,11 @@ export class InteractionOutput implements WoT.InteractionOutput {
 
         const validate = ajv.compile<T>(this.schema);
 
-        // is content type vaild?
-        if (!this.form || !ContentSerdes.get().isSupported(this.content.type)) {
+        // is content type valid?
+        /* if (!this.form || !ContentSerdes.get().isSupported(this.content.type)) {
```

Comment:
This code is needed, see the scripting API algorithm: https://w3c.github.io/wot-scripting-api/#the-value-function step 4

Reply:
applied again